src/lib/components/Cytoscape.react.js

```diff
@@ -953,6 +965,27 @@ Cytoscape.propTypes = {
          */
         target: PropTypes.object,
     }),
+
+    /**
+     * Dictionary specifying configuration options to overlay a leaflet map on top of Cytoscape. All
+     * configuration options are optional; provide an empty dictionary to use default options. Requires
+     * latitude and longitude properties to be included in the node data for positional information.
+     * Requires preset layout to be used. Requires extra layouts to be loaded.
```

Comment:
This is a little confusing alongside `leaflet.preset`. We can either be more explicit in documenting that these are two separate things, or perhaps change `leaflet.preset` to something like `leaflet.provider`? Given that that's what leaflet calls these, perhaps that's cleaner anyway.